Source/Core/DolphinQt/Config/Mapping/MappingCommon.h

```diff
@@ -20,7 +20,8 @@ namespace MappingCommon
 {
 QString GetExpressionForControl(const QString& control_name,
                                 const ciface::Core::DeviceQualifier& control_device,
-                                const ciface::Core::DeviceQualifier& default_device);
+                                const ciface::Core::DeviceQualifier& default_device,
+                                bool quote = true);
```

Comment:
These should likely take an enum value instead of a boolean. This way more info is conveyed at the calling sites than an ambiguous boolean value.

Reply:
That seems a bit heavy-handed for this one simple thing.

Reply:
It isn't though. It's more self-documenting and disambiguates code (and has the benefit of also preventing cases of accidentally passing integral values, as its type is more strongly enforced). Using enums instead of boolean values in APIs isn't uncommon.

Reply:
Fair enough I suppose. Any suggestions for naming?